recipes/dd-fips-proxy.rb

```diff
@@ -0,0 +1,60 @@
+#
+# Cookbook:: datadog
+# Recipe:: dd-fips-proxy
+#
+# Copyright:: 2011-2015, Datadog
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'yaml' # Our erb templates need this
+
+is_windows = platform_family?('windows')
+
+# Install the agent
+if is_windows
+  raise 'Windows currently unsupported for the datadog FIPS proxy.'
+else
+  include_recipe '::_install-fips-proxy-linux'
+end
+
+# Set the Datadog FIPS proxy service enable or disable
+fips_proxy_enable = node['datadog']['fips_proxy_enable'] ? :enable : :disable
+# Set the Datadog FIPS proxy service startup action
+fips_proxy_start = node['datadog']['fips_proxy_start'] ? :start : :stop
```

Comment:
Could we have default values in `attributes/defaulr.rb` for `fips_proxy_enable` and `fips_proxy_start`?